visual_behavior/data_access/loading.py

```diff
@@ -450,19 +517,31 @@ def stimulus_presentations(self):
         if 'orientation' in stimulus_presentations.columns:
             stimulus_presentations = stimulus_presentations.drop(columns=['orientation', 'image_set', 'index'])
         stimulus_presentations = reformat.add_change_each_flash(stimulus_presentations)
+        stimulus_presentations['pre_change'] = stimulus_presentations['change'].shift(-1)
+        stimulus_presentations = reformat.add_epoch_times(stimulus_presentations)
+        stimulus_presentations = reformat.add_mean_running_speed(stimulus_presentations, self.running_speed)
+        stimulus_presentations = reformat.add_licks_each_flash(stimulus_presentations, self.licks)
+        stimulus_presentations = reformat.add_response_latency(stimulus_presentations)
+        stimulus_presentations = reformat.add_rewards_each_flash(stimulus_presentations, self.rewards)
+        stimulus_presentations['licked'] = [True if len(licks) > 0 else False for licks in
+                                            stimulus_presentations.licks.values]
+        stimulus_presentations['lick_rate'] = stimulus_presentations['licked'].rolling(window=320, min_periods=1,
+                                                                                       win_type='triang').mean() / .75
+        stimulus_presentations['rewarded'] = [True if len(rewards) > 0 else False for rewards in
+                                              stimulus_presentations.rewards.values]
+        stimulus_presentations['reward_rate'] = stimulus_presentations['rewarded'].rolling(window=320, min_periods=1,
+                                                                                           win_type='triang').mean()
+        stimulus_presentations = reformat.add_response_latency(stimulus_presentations)
         stimulus_presentations = reformat.add_epoch_times(stimulus_presentations)
         self._stimulus_presentations = stimulus_presentations
         return self._stimulus_presentations
 
     @property
     def extended_stimulus_presentations(self):
-        stimulus_presentations = super().stimulus_presentations.copy()
+        stimulus_presentations = self.stimulus_presentations.copy()
```

Comment:
it will result in extended_stimulus_presentations inheriting all the reformatting / added columns done in the code above to get stimulus_presentations, rather than pulling stimulus_presentations directly from the SDK, which is what the super() does (i think)